MetabolicAtlas/MetabolicAtlas
# fix: indicate when MapsAvailable component is scrollable

**Related issue(s) and PR(s)**  
This PR closes #1114

It solves the problem by adding a small shade as indication of scrollable content in the MapsAvailable component. This is mostly useful for platforms which hide the scrollbars of elements.

**Type of change**  
 
**List of changes made**  
- Add top and bottom indicators of hidden content for the MapsAvailable component

**Screenshot of the fix**  
![scroll_combine](https://user-images.githubusercontent.com/124560665/225321407-6bdb215b-ae07-447f-a601-4405536e19c6.png)

**Testing**  
- Rebuild project
- Make sure the indicators work for: http://localhost/explore/Human-GEM/gem-browser/metabolite/MAM02026c
- Make sure the indicators are missing for: http://localhost/explore/Human-GEM/gem-browser/metabolite/MAM00162c

**Definition of Done checklist**
